imports/ui/components/T9n.js

```diff
@@ -6,7 +6,26 @@ Meteor.startup(function() {
       accounts: {
         "Match failed": "Erro de Validação"
       }
-    }
+    },
```

Comment:
@caike08 we are now translating the html select values here, I think it's a good way to do that.

This is an example: https://github.com/BancoTalentosTCC/BancoTalentosTCC/pull/73/files#diff-8e18eff02ee645797e99a4b17467499cR83